modules/cli/pages/index.adoc

```diff
@@ -7,9 +7,11 @@ include::{mod-loc}attributes.adoc[]
 = Configuring Skupper sites using the CLI
 
 [role="system:abstract"]
-Using the Skupper command-line interface (CLI) allows you to create and manage Skupper sites from the context of the current namespace.
+Using the Skupper command-line interface (CLI) allows you to create and manage {skupper-name} sites from the context of the current namespace.
```

Comment:
Why the first appearance of `Skupper` is literal and the second uses the `{skupper-name}` variable?